DDDEastAnglia/Controllers/HomeController.cs

```diff
@@ -9,15 +9,17 @@ namespace DDDEastAnglia.Controllers
     public class HomeController : Controller
     {
         private readonly IConferenceLoader conferenceLoader;
+        private readonly SponsorModelQuery sponsorModelQuery;
 
-        public HomeController(IConferenceLoader conferenceLoader)
+        public HomeController(IConferenceLoader conferenceLoader, SponsorModelQuery sponsorModelQuery)
```

Comment:
Ahh no probs.. was trying to re-use what we'd already done.. 

Have you looked at the existing IQuery? Is that what you're suggesting we use?

Question.. 

What should the query return? ViewModels DomainModels, Storage Models?

I'm also confused on what we're trying to do here.. 

Perhaps an example of something in the existing code base which does this correctly would help? Is there one?

Any hints most welcome :+1: